chapters/fa/chapter2/1.mdx

```diff
@@ -0,0 +1,20 @@
+#مقدمه
```

Comment:
To render this on the website, we should add a `_toctree.yml` file with the following content:

```
- title: 1. Transformer models # Translate this
  sections:
  - local: chapter2/1
    title: Introduction # And this
```